src/Loader.php

```diff
@@ -1099,6 +1099,7 @@ public static function add_component_settings( $settings ) {
 		$settings['wcAdminAssetUrl'] = plugins_url( 'images/', dirname( __DIR__ ) . '/woocommerce-admin.php' );
 		$settings['wcVersion']       = WC_VERSION;
 		$settings['siteUrl']         = site_url();
+		$settings['shopUrl']         = get_permalink( woocommerce_get_page_id( 'shop' ) );
```

Comment:
Hi there 👋
I think that the method `woocommerce_get_page_id` has been deprecated since version 3.0. I'll create a follow-up PR to use the recommended replacement  (`wc_get_page_id`) instead.